templates/corporate/upgrade.html

```diff
@@ -3,7 +3,7 @@
 
 {% block customhead %}
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<script src="https://checkout.stripe.com/checkout.js"></script>
+<script src="https://js.stripe.com/v3/"></script>
```

Comment:
Stripe has a [`@stripe/stripe-js` NPM package](https://www.npmjs.com/package/@stripe/stripe-js) that exposes an ES module and TypeScript declarations.

Reply:
Thanks. Will make use of it.